### PR TITLE
feat: add method to generate url with parameters

### PR DESCRIPTION
**Pull request by ZEROM22**

### CODE REVIEW

File: app/client.test.js [hunk 1]
```diff
@@ -26,6 +26,30 @@ test("instance creation and initialization", () => {
...
 })
 
+test("url generate a valid URL with filters", () => {
+  // @ts-ignore
+  const client = new Client("https://example.com/", () => {})
+  const filters = {
+    count: 1,
+    sortBy: "DateAndTime",
+    sortOrder: "descending"
+  }
```
Comment (vanyauhalin) [inline app/client.test.js:36]: Please add a type for the `filters` object and add all of the possible properties.

File: app/client.js [hunk 2]
```diff
@@ -16,16 +16,51 @@ class Client {
   }
 
   /**
-   * @param {HttpMethod} method
+   * @typedef {Object} Filters
```
Comment (vanyauhalin) [inline app/client.js:19]: Please add the `Filters` object to the `index.d.js`.

**Review by vanyauhalin — CHANGES_REQUESTED** [review]

File: app/client.test.js (hunk 1 above)
Comment (ZEROM22) [reply app/client.test.js]: > Please add a type for the `filters` object and add all of the possible properties.

I've added all the filters [listed here](https://api.onlyoffice.com/docspace/filters).

File: app/client.js (hunk 2 above)
Comment (ZEROM22) [reply app/client.js]: > Please add the `Filters` object to the `index.d.js`.

Added.

**Review by vanyauhalin — APPROVED** [review]

**Pull request merged**